Makefile

```diff
@@ -23,11 +29,27 @@ NETWORK_ISOLATION   ?= true
 # options to pass in all targets that use git clone
 GIT_CLONE_OPTS      ?=
 
+# set to 3 to use a 3-node galera sample
+GALERA_REPLICAS         ?=
+
 # OpenStack Operator
 OPENSTACK_IMG                ?= quay.io/openstack-k8s-operators/openstack-operator-index:latest
 OPENSTACK_REPO               ?= https://github.com/openstack-k8s-operators/openstack-operator.git
 OPENSTACK_BRANCH             ?= main
-OPENSTACK_CTLPLANE           ?= $(if $(findstring true,$(NETWORK_ISOLATION)),config/samples/core_v1beta1_openstackcontrolplane_network_isolation.yaml,config/samples/core_v1beta1_openstackcontrolplane.yaml)
+ifeq ($(NETWORK_ISOLATION), true)
+ifeq ($(DBSERVICE), galera)
+OPENSTACK_CTLPLANE           ?= core_v1beta1_openstackcontrolplane_galera_network_isolation.yaml
```

Comment:
This needs to be prepended with `config/samples/`

Reply:
Oops, thanks! Fixed.